src/scheduler/stride.rs

```diff
@@ -13,33 +14,55 @@ pub struct StrideScheduler {
 pub struct StrideSchedulerInner {
     max_time_slice: usize,
     infos: Vec<StrideProcInfo>,
-    queue: BinaryHeap<(Stride, Tid)>, // It's max heap, so pass < 0
+    queue: BinaryHeap<Reverse<(Stride, Tid)>>, // It's max heap, so use Reverse
 }
 
 #[derive(Debug, Default, Copy, Clone)]
 struct StrideProcInfo {
-    present: bool,
+    valid: bool,    // Whether it is exit
+    present: usize, // Times it is pushed
```

Comment:
Grammar

Reply:
Why is it needed

Reply:
Thought again and I realized that it is needed, because the invalid ProcInfo would be poped.